k8s-install/1.6/canal.yaml

```diff
@@ -0,0 +1,211 @@
+# This ConfigMap can be used to configure a self-hosted Canal installation.
+# See `canal.yaml` for an example of a Canal deployment which uses
+# the config in this ConfigMap.
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: canal-config
+  namespace: kube-system
+data:
+  # The interface used by canal for host <-> host communication.
+  # If left blank, then the interface is chosing using the node's
+  # default route.
+  canal_iface: "eth1"
```

Comment:
Why is this eth1? the other manifest leaves it blank. I just tried on gcloud and my hosts don't have an eth1 so i receive this error:
```
turk@turk-tf-kubecoreos-master ~ $ kubectl logs canal-rwqc9 -n kube-system kube-flannel
I0428 21:59:58.656728       1 kube.go:111] Waiting 10m0s for node controller to sync
I0428 21:59:58.656793       1 kube.go:315] starting kube subnet manager
I0428 21:59:59.656932       1 kube.go:118] Node controller sync successful
I0428 21:59:59.657003       1 main.go:132] Installing signal handlers
E0428 21:59:59.657268       1 main.go:152] Failed to create NetworkManager: error looking up interface eth1: route ip+net: no such network interface

```